examples/plot_simulate_evoked_newapi.py

```diff
@@ -0,0 +1,207 @@
+"""
+====================================
+Simulate dipole using new drives-API
+====================================
+
+This example demonstrates how to simulate a dipole for evoked-like
+waveforms using HNN-core.
+
+Note that the output will be slightly different from HNN-GUI due to different
+"seeds" being used when creating the random exogeneous input spikes to the
+network. The results should match qualitatively, however.
+"""
+
+# Authors: Mainak Jas <mainak.jas@telecom-paristech.fr>
+#          Sam Neymotin <samnemo@gmail.com>
+#          Blake Caldwell <blake_caldwell@brown.edu>
+#          Christopher Bailey <cjb@cfin.au.dk>
+
+import os.path as op
+import tempfile
+
+###############################################################################
+# Let us import hnn_core
+
+import hnn_core
+from hnn_core import simulate_dipole, read_params, Network, read_spikes
+from hnn_core.viz import plot_dipole
+
+hnn_core_root = op.dirname(hnn_core.__file__)
+
+###############################################################################
+# Then we read the parameters file
+params_fname = op.join(hnn_core_root, 'param', 'default.json')
+params = read_params(params_fname)
+print(params)
+
+###############################################################################
+# This is a lot of parameters! We can also filter the
+# parameters using unix-style wildcard characters
+print(params['L2Pyr_soma*'])
+
+###############################################################################
+# Let us first create our network from the params file and visualize the cells
+# inside it. The default behaviour of Network is to add and instantiate six
+# 'default' drives, but we will suppress that by setting the
+# `initialise_hnn_drives`-argument to `False`.
+net = Network(params, initialise_hnn_drives=False)
+net.plot_cells()
+
+###############################################################################
+# The network of cells is now defined, to which we add external drives as
+# required. Weights are prescribed separately for AMPA and NMDA receptors
+# (receptors that are not used can be omitted or set to zero)
+
+# Two "silent" bursty drives (not connected, but affect seeds via gid assgnmt)
+for ii in range(2):
+    weights_ampa = {}
+    weights_nmda = {}
+    dispersion_time = 0.1
+    net.add_bursty_drive(
+        f'bursty{ii + 1}', distribution='normal', t0=0, sigma_t0=1., T=1.,
```

Comment:
Thoughts for a future ENH, what do you think about appending numbers to the drive name internally if the same `str` is passed multiple times? 

I'm slightly against it since it may lead to unexpected behavior, but you guys have been thinking about what's intuitive for this API for awhile now.

Reply:
Removed in 4e4ad61